src/components/MobileDrawer.css

```diff
@@ -0,0 +1,3 @@
+.AboutMe {
```

Comment:
why is this not in-line like the rest?

Reply:
The MUI component library I'm currently using will not let certain CSS properties be changed in-line so I had to use a style-sheet with a class defined to get this specific property to change. I'm not entirely sure why this is or why the style-sheet + class declaration works, my best guess is because of specificity.